content/events/hacktoberfest.adoc

```diff
@@ -9,18 +9,19 @@ tags:
   - programs
   - hacktoberfest
 links:
-  gitter: jenkinsci/hacktoberfest
   discourse: https://community.jenkins.io/t/hacktoberfest-2021-join-us/375
 opengraph:
-  image: /images/hacktoberfest/hacktoberfest_2021_social.png
+  image: /images/hacktoberfest/hacktoberfest_2022_social.png
 ---
 
-image:/images/hacktoberfest/logo-hacktoberfest-2021-full.svg[Hacktoberfest, role=center]
+image:/images/hacktoberfest/hacktoberfest_2022.svg[Hacktoberfest, role=center]
 
-NOTE: Hacktoberfest 2021 is coming soon! We are going to participate as the Jenkins community,
+NOTE: Hacktoberfest 2022 is coming soon! We are going to participate as the Jenkins community,
 and we are looking for contributors and maintainers who want to join us in October!
 See link:https://community.jenkins.io/t/hacktoberfest-2021-join-us/375[this conversation] for more information and Q&A.
 
+//TODO: fix the discourse link (see also header) as soon as available
```

Comment:
do we need a new link? I know it said 2021 before, but really it could be the same link.

Reply:
Now that the PR is merged, I can create the topic.
Some kind of "chicken & egg" situation

Will update the Hacktoberfest data accordingly along with de-drafting the blog post